src/settings/MappingProfiles/MappingProfileDetails/MappingProfileDetails.js

```diff
@@ -0,0 +1,196 @@
+  };
+
+  return (
+    <FormattedMessage id="ui-data-export.mappingProfiles.newProfile">
+      {contentLabel => (
+        <Layer
+          isOpen
+          contentLabel={contentLabel}
+        >
+          <FullScreenView
+            id="mapping-profile-details"
+            paneTitle={record.name}
+            onCancel={onCancel}
+          >
+            {!hasLoaded
+              ? <Preloader />
+              : (
+                <div className={css.viewMappingProfileContent}>
+                  <AccordionStatus>
+                    <Row>
+                      <Col xs={9}>
+                        <Headline
+                          data-test-headline
+                          size="x-large"
+                          tag="h2"
+                          margin="small"
+                        >
+                          {record.name}
+                        </Headline>
+                      </Col>
+                      <Col xs={3}>
+                        <Row end="xs">
+                          <Col xs>
+                            <ExpandAllButton />
+                          </Col>
+                        </Row>
+                      </Col>
+                    </Row>
+                    <AccordionSet id="mapping-profile-details-accordions">
+                      <Accordion label={<FormattedMessage id="ui-data-export.summary" />}>
+                        <MetaSection
+                          createdBy={record.metadata.createdByUserId}
+                          lastUpdatedBy={record.metadata.updatedByUserId}
+                          createdDate={record.metadata.createdDate}
+                          lastUpdatedDate={record.metadata.updatedDate}
+                        />
+                        <Row>
+                          <Col xs>
+                            <KeyValue
+                              data-test-mapping-profile-name
+                              label={<FormattedMessage id="ui-data-export.name" />}
+                              value={record.name}
+                            />
+                          </Col>
+                        </Row>
+                        <Row>
+                          <Col xs>
+                            <KeyValue
+                              data-test-mapping-profile-folio-record-type
+                              label={<FormattedMessage id="stripes-data-transfer-components.folioRecordType" />}
+                            >
+                              <List
+                                items={record.recordTypes}
+                                itemFormatter={recordType => (
+                                  <li>
+                                    <FormattedMessage id={FOLIO_RECORD_TYPES[recordType].captionId} />
+                                  </li>
+                                )}
+                                isEmptyMessage={<NoValue />}
+                                listClass={css.list}
```

Comment:
I suggest to add more accurate name. For example, recordTypesList or something else. @Vladyslav-Velytskyi what do you think about it?

Reply:
I'm agree with you, nice suggestion